daemon/openrazer_daemon/hardware/monitor.py

```diff
@@ -0,0 +1,20 @@
+"""
+Monitor class
+"""
+
+from openrazer_daemon.hardware.device_base import RazerDeviceBrightnessSuspend as _RazerDeviceBrightnessSuspend
+
+
+class RazerBaseStationChroma(_RazerDeviceBrightnessSuspend):
```

Comment:
Spotted a typo - needs a new class name
```suggestion
class RazerRaptor27Inch(_RazerDeviceBrightnessSuspend):
```

Reply:
On second thoughts (sorry!), perhaps this should just be `RazerRaptor27` to keep it consistent with the naming for Blade models (and driver files)?

Reply:
Sure - I struggled with the name too.

Everyone expects a 4K version to be launched imminently, so I was initially tempted to call it `RazerRaptor1440` but I thought that was unwieldy...

I'll change it to just `RazerRaptor27` for now, and it can be retroactively changed if/when the range grows.